src/in_process_server.cpp

```diff
@@ -308,17 +308,7 @@ class wlcs::Client::Impl
         wl_shell_surface_set_toplevel(shell_surface);
         wl_surface_commit(surface);
 
-        auto buffer = std::make_shared<ShmBuffer>(client, width, height);
-
-        wl_surface_attach(surface, *buffer, 0, 0);
-
-        /*
-         * We can't drive buffer cleanup by the buffer.release() event, as that's not
-         * guaranteed to be sent in the case of error or server shutdown.
-         *
-         * Just keep the buffers alive for the lifetime of the client.
-         */
-        client_buffers.push_back(buffer);
```

Comment:
Doesn't this make client_buffers unused? (Except for a clear() in the destructor.)

Reply:
resolved.